modules/frontend/variables.tf

```diff
@@ -30,12 +30,18 @@ variable "name" {
 }
 
 variable "network" {
-  description = "VPC network for the forwarding rule. It should not be default"
+  description = "VPC network for the forwarding rule. It should not be default. The VPC network should also have only one REGIONAL_MANAGED_PROXY subnetwork in the same region as of this regional load balancer. Please go to the subnets tab of your VPC network and check if a REGIONAL_MANAGED_PROXY subnet exists under `Reserved proxy-only subnets for load balancing` section. If the REGIONAL_MANAGED_PROXY doesn't exists, set create_proxy_only_subnet parameter to provision it as part of this component deployment."
   type        = string
 }
 
+variable "create_proxy_only_subnet" {
+  description = "Create a REGIONAL_MANAGED_PROXY subnetwork in the provide VPC network."
+  type        = bool
+  default     = false
```

Comment:
What error do we get if a proxy only subnet does not exist? I am trying to understand if the default behavior as true would be more helpful or if false would be better.

Reply:
Forwarding rule creation fails without proxy only subnet. 

`Error: Error creating ForwardingRule: googleapi: Error 400: Invalid value for field 'resource.target': 'https://www.googleapis.com/compute/v1/projects/ci-int-regional-lb-http-bd31/regions/us-central1/targetHttpProxies/frontend-lb-regional-http-proxy'. An active proxy-only subnetwork is required in the same region and VPC as the forwarding rule., invalid`

If we set to `true`, the proxy-only subnet creation fails because only one can be present per region. This is the current behavior. I believe we want to avoid this. Also with default `true`, users would need to set this to `false` explicitly for all but one application using same vpc.

Reply:
Yes in that case we can keep as false. I just wanted to ensure that the message was actionable enough, thanks for checking!